galaxy.yml

```diff
@@ -2,7 +2,7 @@
 
 namespace: community
 name: hashi_vault
-version: 3.1.0
+version: 3.0.1
```

Comment:
Deprecations must appear in minor releases.

Reply:
oh interesting, I don't think I ever realized that, thanks!

Reply:
See the semver spec, point 7: https://semver.org/

> Minor version Y (x.Y.z | x > 0) [...] MUST be incremented if any public API functionality is marked as deprecated.

Reply:
I see, I guess I sort of felt like we're using the deprecation system sometimes as an announcement system for breaking changes, even when it's not really deprecating any functionality, so it didn't register with me based on that description, that a change of a default value would count. But I have no issue with it, good to know, ty!

Reply:
For announcement-only deprecations (that have no practical effect on users since there is no code change) I think bugfix releases are totally fine, even though it might be strictly against semver :)